iothub/device/samples/PnpDeviceSamples/readme.md

```diff
@@ -0,0 +1,17 @@
+### Plug and Play Samples
+
+These samples demonstrate how a plug and play enabled device interacts with IoT hub, to:
+- Send telemetry.
+- Update read-only and read-write porperties.
+- Respond to command invocation. 
+
+The samples demonstrate two scenarios:
+- The device is modeled as a plug and play device, having only a root interface - [Thermostat][d-thermostat]
```

Comment:
I would call out the sample name in beginning with a link to interface followed at the end both here and on line 11